src/Transition/CreateNewIDFUsingRulesV23_2_0.f90

```diff
@@ -493,6 +493,30 @@ SUBROUTINE CreateNewIDFUsingRules(EndOfFile,DiffOnly,InLfn,AskForInput,InputFile
                 OutArgs(23:CurArgs+4)=InArgs(19:CurArgs)
                 CurArgs = CurArgs + 4
 
+              CASE('COIL:COOLING:DX:CURVEFIT:PERFORMANCE')
```

Comment:
I thought I was going to be merging this right now but this is another big set of transitions for the coils.  I need to look this over deeper and ... probably?  ... do some manual merging of these rules.

Reply:
> This is still not an integer. Did you by chance forget to push a commit?

Let me see. I felt some of my earlier commit got lost somehow. I remember changing the field name once and realize it still looks like the old one so I changed it again. I will change it to integer again.